scripts/tests/api_compare/.env

```diff
@@ -4,4 +4,6 @@ LOTUS_IMAGE=filecoin/lotus-all-in-one:v1.26.0-rc3-calibnet
 FIL_PROOFS_PARAMETER_CACHE=/var/tmp/filecoin-proof-parameters
 LOTUS_RPC_PORT=1234
 FOREST_RPC_PORT=2345
+FOREST_OFFLINE_RPC_PORT=2346
```

Comment:
Does it have to be `2346`? I wanted to use it for healthcheck endpoints. :rofl:

Reply:
Changed to `3456`